quickstarts/hello-rules-multi-module/README.md

```diff
@@ -67,6 +50,15 @@ If the `rhdm-kieserver-rhel8` is not found, install it under the 'openshift' nam
 $ oc create -f https://raw.githubusercontent.com/jboss-container-images/rhdm-7-openshift-image/master/rhdm79-image-streams.yaml -n openshift
 ```
 
+Deploy the `credentials secret` provided as example:
+
+```bash
+$ oc create -f https://raw.githubusercontent.com/jboss-container-images/rhdm-7-openshift-image/master/example-credentials.yaml
```

Comment:
When I opened link there are admin user name and admin password comment switched, it would be good to fix them there as well as comment at [line 60](https://github.com/jboss-container-images/rhdm-7-openshift-image/pull/365/files#diff-d9c39646c85379897fccc7767408b615R60) may not be enough.